transifex/native/cds.py

```diff
@@ -288,6 +316,7 @@ def _get_headers(self, use_secret=False, etag=None):
                 secret=(':' + self.secret if use_secret else '')
             ),
             'Accept-Encoding': 'gzip',
+            'Accept-version': 'v2',
```

Comment:
Minor: I think it's convention to have the headers capitalized, so `Accept-Version`

Reply:
Fixed